core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -138,6 +146,40 @@ public Void call() throws S3Exception {
     });
   }
 
+  /**
+   * @summary gets a bucket and lists all the objects in it
+   * @param bucket the bucket name
+   * @return the response object
+   */
+  @GET
+  @Path(BUCKET_PARAM)
+  @ReturnType("ListBucketResult")
+  // TODO(chaomin): consider supporting request params like max-keys, prefix, delimiter and
```

Comment:
what is the reason for not supporting these now?. i would expect most application to use prefix and delimeter